javav2/example_code/elasticbeanstalk/src/main/java/com/aws/example/DescribeApplications.java

```diff
@@ -0,0 +1,74 @@
+//snippet-sourcedescription:[DescribeApplication.java demonstrates how to describe an AWS Elastic Beanstalk application.]
+//snippet-keyword:[SDK for Java v2]
+//snippet-keyword:[Code Sample]
+//snippet-service:[AWS Elastic Beanstalk ]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[03/10/2022]
+//snippet-sourceauthor:[scmacdon - aws]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+
+package com.aws.example;
+
+//snippet-start:[eb.java2.describe_app.import]
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.elasticbeanstalk.ElasticBeanstalkClient;
+import software.amazon.awssdk.services.elasticbeanstalk.model.DescribeApplicationsResponse;
+import software.amazon.awssdk.services.elasticbeanstalk.model.ApplicationDescription;
+import software.amazon.awssdk.services.elasticbeanstalk.model.DescribeEnvironmentsRequest;
+import software.amazon.awssdk.services.elasticbeanstalk.model.DescribeEnvironmentsResponse;
+import software.amazon.awssdk.services.elasticbeanstalk.model.EnvironmentDescription;
+import software.amazon.awssdk.services.elasticbeanstalk.model.ElasticBeanstalkException;
+import java.util.List;
+//snippet-end:[eb.java2.describe_app.import]
+
+
+/**
+ * To run this Java V2 code example, ensure that you have set up your development environment, including your credentials.
+ *
+ * For information, see this documentation topic:

```

Comment:
For more information, see the following documentation topic:

Reply:
fixed